### PR TITLE
Add labels for a better manipulation

### LINKED ISSUE #6: Define a common label on all objects
A common label can be used to manipulate (e.g. delete) all objects defined in this template

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves #6 

* define common labels
* move app before labels

### CODE REVIEW

**Comment by pavolloffay** [comment]
@ruromero could you please review?

File: es-cluster-deployment.yml [hunk 1]
```diff
@@ -13,7 +14,7 @@ metadata:
       }]
   labels:
     app: elasticsearch
-  name: elasticsearch
+    elasticsearch: elasticsearch-statefulset
```
Comment (ruromero) [inline es-cluster-deployment.yml]: I don't see much benefit of having this compared to app=elasticsearch
If you want to get only this resource (the only sts) you'll do `oc get sts -l app=elasticsearch`

File: es-cluster-deployment.yml [hunk 2]
```diff
@@ -25,6 +26,7 @@ spec:
     metadata:
       labels:
         app: elasticsearch
+        elasticsearch: elasticsearch-pod
```
Comment (ruromero) [inline es-cluster-deployment.yml]: same as above

File: es-cluster-deployment.yml [hunk 3]
```diff
@@ -115,3 +121,5 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: elasticsearch
+  labels:
+    elasticsearch: elasticsearch-serviceaccount
```
Comment (ruromero) [inline es-cluster-deployment.yml]: instead of having this, make sure all the resources have
```
labels:
   app: elasticsearch
```

**Comment by pavolloffay** [comment]
Changes applied, you are right, now it's better to stick with `app=elasticsearch`. I have added that label to all resources.

File: es-cluster-deployment.yml (hunk 1 above)
Comment (ruromero) [reply es-cluster-deployment.yml]: You missed this one ;)

Comment (pavolloffay) [reply es-cluster-deployment.yml]: darm, now fixed

**Pull request merged**